forem/forem
# Allow option for "no readme" with GitHub embeds

## What type of PR is this? (check all applicable)


## Description
Prior to unified embeds, when embedding a Github repository URL, you could choose not to display the repo's README in the embed by passing the `no-readme` option. This PR allows for that option to be passed using the `{% embed ... %}` method.

## Related Tickets & Documents
closes https://github.com/forem/forem/issues/16673


## QA Instructions, Screenshots, Recordings
Grab the URL of a Github repository (like https://github.com/forem/forem). Create an embed with this URL and add `no-readme` or `noreadme` after the URL. The embed should render successfully with no readme displayed.

### UI accessibility concerns?
None; backend changes only

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?
I think it could also be helpful to share this change in one of the general RoundUp posts, incase Forem users were missing it for their use.

![K. Michelle asking if someone is reading her](https://media.giphy.com/media/1nQ6bcq95YJ0zgmYFR/giphy-downsized.gif)
